Manifest.toml

```diff
@@ -0,0 +1,426 @@
+# This file is machine-generated - editing it directly is not advised
```

Comment:
We're still using REQUIRE until METADATA/attobot updates.

Reply:
I haven't gotten rid of the REQUIRE file, and they aren't mutually exclusive. But the manifest is useful for getting a known good version of things if I end up on e.g. master branches of Flux. Happy to remove before we merge anything if you prefer, though.

Reply:
This just leads to issues because CI doesn't actually test the REQUIRE file. For example, in this PR currently Flux isn't in the REQUIRE which should error, but since it uses the Project/Manifest this issue goes undetected. So it's safer to just not use this at all until it's actually what's used by METADATA.